fluent-bit.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: fluent-bit
+  version: 2.0.8
+  epoch: 0
+  description: Fast and Lightweight Log processor and forwarder
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
```

Comment:
Do we need to do something for this btw?

Reply:
they are left TODO like this for now.

@rawlingsj what should we put here for attestation?

Reply:
Ok, I just wanted to check if it needed anything upstream

Reply:
the attestation should ideally be a collection of the copyright lines plus the "this software is licensed under blabla" text.  similar to `debian/copyright`, but we are still kinda playing with the format, so it's not so important right now.

Reply:
Filed https://github.com/wolfi-dev/os/issues/510 to track